tools/intersmash-tools-core/src/main/java/org/jboss/intersmash/tools/junit5/IntersmashExtension.java

```diff
@@ -52,14 +52,6 @@ public class IntersmashExtension implements BeforeAllCallback, AfterAllCallback,
 
 	private static final Namespace NAMESPACE = Namespace.create("org", "jboss", "intersmash", "IntersmashExtension");
 	private static final String INTERSMASH_SERVICES = "INTERSMASH_SERVICES";
-	//	private OpenShiftRecorderService openShiftRecorderService;
```

Comment:
If there is some reason for keeping this commented code, can you please describe it in java comment?

Reply:
I think I've removed it, just because it was actually not needed now.
We've got enough information logged by the OCP CI infra checks.

Reply:
Ah, my mistake, sorry